Hmm, cannot publish

For some reason, cannot publish, even after updating tokens

```
npm ERR! code ENEEDAUTH
npm ERR! need auth auth required for publishing
npm ERR! need auth You need to authorize this machine using `npm adduser`
```

--- a/circle.yml
+++ b/circle.yml
@@ -6,4 +6,4 @@ test:
     - npm test
   post:
     - ./refs.sh || true
-    - DEBUG=condition npm run semantic-release || true
+    - DEBUG=condition npx semantic-release || true

--- a/package.json
+++ b/package.json
@@ -48,7 +48,7 @@
   },
   "devDependencies": {
     "pre-git": "3.16.0",
-    "semantic-release": "^8.2.0",
+    "semantic-release": "11.0.1",
     "standard": "10.0.3"
   },
   "config": {

--- a/package.json
+++ b/package.json
@@ -6,10 +6,9 @@
   "scripts": {
     "test": "node .",
     "size": "t=\"$(npm pack .)\"; wc -c \"${t}\"; tar tvf \"${t}\"; rm \"${t}\";",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "lint": "standard --verbose --fix src/*.js",
     "pretest": "npm run lint",
-    "test-self": "CIRCLECI=true CIRCLE_BRANCH=master DEBUG=condition npm_config_loglevel=verbose npm run semantic-release",
+    "test-self": "CIRCLECI=true CIRCLE_BRANCH=master DEBUG=condition npm_config_loglevel=verbose npx semantic-release",
     "commit": "commit-wizard"
   },
   "files": [

